tests/features/cli.feature

```diff
@@ -0,0 +1,17 @@
+# Date:  08/08/2020
+# Author: Laurentiu Mandru
+# Email: mclaurentiu79@gmail.com
+
+Feature: CLI for report builder
+
+  The CLI produces the desired output
+
+  Scenario Outline: Generate a report from a template
```

Comment:
very good simple and concise scenario. Perhaps we can extend the features and use cases latter.